kimchi/src/tests/framework.rs

```diff
@@ -64,6 +65,12 @@ impl TestFramework {
         self
     }
 
+    #[must_use]
+    pub(crate) fn num_prev_challenges(mut self, num_prev_challenges: usize) -> Self {
```

Comment:
humm... I did it differently in https://github.com/o1-labs/proof-systems/pull/609/files#diff-1521f525c884d89a65ff41457738a87b33184b84f40d367b6dc9c997ed926d6cR69